app/web/src/components/ProjectCell/ProjectCell.tsx

```diff
@@ -26,8 +26,14 @@ export const QUERY = gql`
         mainImage
         createdAt
         updatedAt
-        userId
         cadPackage
+        forkedFrom {
+          id
+          title
+          user {
+            userName
```

Comment:
I can't remember if the formatting for the forks always looked like this in the designs or not
![image](https://user-images.githubusercontent.com/29681384/135072198-362c13f1-0ae5-4337-9c6f-e44b3e5dfeea.png)
So I'll ask, are you querying for the userName because thats how you want to fork formatted, <title> by <user> or because you need the username to link to a project?